gulp/modules/ComponentHelper.js

```diff
@@ -48,9 +47,10 @@ var ComponentSamplesHelper = function() {
             })))
             .pipe(gulp.dest(destFolder))
             .pipe(Plugins.rename(componentName + '.min.css'))
-            .pipe(Plugins.cssMinify({
-              safe: true
-            }))
+            // .pipe(Plugins.cssMinify({
```

Comment:
If this code isn't needed, please delete the commented-out stuff. If there's a reason to keep it around (i.e. for future reference), please add a comment. But it doesn't look like it's needed?

Reply:
I'll pop it out, thanks! Yes, it's not needed since we changed over to the gulp-clean-css dependency there is no `safe: true` option